src/components/SignInForm/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react' 
+import Button from 'react-bootstrap/Button' 
+
+import { Formik, Field, Form, ErrorMessage } from 'formik' 
+import validationSchema from './validationSchema'
+
+function SignInForm ({onSubmit, mutationError}){
+
+  return(
+
+    <Formik
+      initialValues={{ username: '', password: ''}}
+      validationSchema={validationSchema}
+      onSubmit={onSubmit}
+    >
+      <Form id= 'form'>
+        <div>
+          <Field name= 'username' type= 'text' className= 'form-input' placeholder= 'Username' />
```

Comment:
No need for spaces between a tag's attribute and its value:

```suggestion
          <Field name='username' type='text' className='form-input' placeholder='Username' />
```

Reply:
Please correct that on all other files.